### DISCUSSION TITLE
EE%40Stanford/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#187] [Announcements]
# EE%40Stanford/

https://opencs.app/EE%40Stanford/

**Comment by yuhengcai1** [comment]
大S左的离谱，宁要211女生不要清北顶会，美本带system顶会但是是男生的话也被拒

**Reply by kmp1001** [reply]
真是太离谱了

**Comment by sofyc** [comment]
23fall thu ee一共录取了5个同学，全是男生，gpa都是年级前10左右水平，暂时没有听说有同届女生申请这个项目

**Comment by xichenpan** [comment]
overall看起来今年录的陆本还是多多了

**Comment by FeijiangHan** [comment]


**Reply by liyanluo1** [reply]
请问这里指的低gpa是多少

**Comment by JamesSand** [comment]
有朋友知道这个项目和 ICME MS 的 bar 哪个更高吗

**Reply by WayneJin0918** [reply]
ICME要高蛮多的
